im3components/wrf_tell/wrf_to_tell_part_two.py

```diff
@@ -0,0 +1,149 @@
+# second step in a processing chain to go from the WRF output files to the input
+# files needed for TELL. All times are in UTC. Missing values are reported as -9999.
+
+# Import all of the required libraries and packages:
+import glob
+import numpy as np
+import pandas as pd
+import os
+import datetime
+from datetime import datetime as dt
+import re
+
+# Set a time variable to benchmark the run time:
+begin_time = datetime.datetime.now()
+
+# Set the data input and output directories:
+data_input_dir = '/Users/burl878/OneDrive - PNNL/Documents/IMMM/Data/TELL_Input_Data/forward_execution/Climate_Forcing/County_Output_Files/'
+data_output_dir = '/Users/burl878/OneDrive - PNNL/Documents/IMMM/Data/TELL_Input_Data/forward_execution/Climate_Forcing/BA_Output_Files/'
+ba_geolocation_input_dir = '/Users/burl878/OneDrive - PNNL/Documents/IMMM/Data/TELL_Input_Data/inputs/Utility_Mapping/CSV_Files/'
+population_input_dir = '/Users/burl878/OneDrive - PNNL/Documents/IMMM/Data/TELL_Input_Data/inputs/'
+
+# Set the year to process:
+year_to_process = '2019';
+
+# Create a list of all of the WRF output files in the "data_input_dir":
+list_of_files = sorted(glob.glob(os.path.join(data_input_dir + year_to_process + '*_County_Mean_Meteorology.csv')))
+
+# Load in the BA service territory map and simplify the data frame:
+ba_service_territory_filename = ba_geolocation_input_dir + 'fips_service_match_' + year_to_process + '.csv'
+col_names = ['county_fips','county_name','ba_number','ba_abbreviation']
+ba_mapping = pd.read_csv(ba_service_territory_filename,index_col=None,header=0)
+ba_mapping = ba_mapping[col_names].copy(deep=False)
+ba_mapping.rename(columns={"county_fips":"County_FIPS",
+                           "county_name":"County_Name",
+                           "ba_number":"BA_Number",
+                           "ba_abbreviation":"BA_Code"},inplace=True)
+ba_mapping = ba_mapping.sort_values("BA_Number")
+ba_mapping = ba_mapping.drop_duplicates()
+del col_names,ba_service_territory_filename
+
+# Load in the population data and simplify the data frame:
+population = pd.read_csv(population_input_dir + '/county_populations_2000_to_2019.csv')
+col_names = ['county_FIPS','pop_'+year_to_process]
+population = population[col_names].copy(deep=False)
+population.rename(columns={"county_FIPS":"County_FIPS",
+                           "pop_"+year_to_process:"Population"},inplace=True)
+del col_names
+
+# Merge the ba_mapping and population data frames together:
+mapping_df = ba_mapping.merge(population, on=['County_FIPS'])
+mapping_df = mapping_df.sort_values("BA_Number")
+mapping_df['Population_Sum'] = mapping_df.groupby('BA_Code')['Population'].transform('sum')
+mapping_df['Population_Fraction'] = mapping_df['Population'] / mapping_df['Population_Sum']
+mapping_df = mapping_df.dropna()
+del population,ba_mapping
+
+# Initialize a set of empty pandas data frames:
+Time = pd.DataFrame()
```

Comment:
Meet PEP 8 standards for variable name style:  https://www.python.org/dev/peps/pep-0008/#function-and-variable-names